invoice.sty loads the fp package with \input instead of \RequirePackage
This creates problems, see
http://tex.stackexchange.com/questions/133566/combining-invoice-and-datatool-to-create-serial-invoices
http://tex.stackexchange.com/questions/162446/problem-using-invoice-and-wallpaper-packages-together
Fix that.
BTS: #740291
---
 texmf-dist/tex/latex/invoice/invoice.sty |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- texlive-extra-2013.20140215.orig/texmf-dist/tex/latex/invoice/invoice.sty
+++ texlive-extra-2013.20140215/texmf-dist/tex/latex/invoice/invoice.sty
@@ -149,7 +149,7 @@
 \RequirePackage{ifthen}
 \RequirePackage{longtable}
 \RequirePackage{calc}
-\input{fp}
+\RequirePackage{fp}
 \input{invoice.def}
 %
 \newcommand{\InvoiceVersion}{0.9}%
